### ISSUE TITLE
Need to fix DOM prices table

### ISSUE THREAD

**Issue by Tim55667757** [Tim55667757/TKSBrokerAPI#81]
```
$ tksbrokerapi -t TRUR --price --depth 10

TKSBrokerAPI.py     L:1209 INFO    [2022-11-11 16:33:27,795] Current prices in order book:

Orders book actual at [2022-11-11 13:33:27] (UTC)
Ticker: [TRUR], FIGI: [BBG000000001], Depth of Market: [10]
----------------------------------------
 Orders of Buyers   | Orders of Sellers
----------------------------------------
 Sell prices (vol.) | Buy prices (vol.)
----------------------------------------
                    | 5.71 (1158)
                    | 5.7 (94330)
                    | 5.69 (112074)
                    | 5.68 (11901)
                    | 5.67 (105064)
                    | 5.66 (23021)
                    | 5.65 (1439671)
                    | 5.64 (22710)
                    | 5.63 (388726)
                    | 5.62 (3237646)
     5.61 (7678909) |
      5.6 (3473245) |
     5.59 (2226084) |
     5.58 (1170453) |
      5.57 (685540) |
5.5600000000000005 (518561) |
       5.55 (55772) |
        5.54 (9274) |
        5.53 (5603) |
     5.52 (1110580) |
----------------------------------------
Total sell: 16934021 | Total buy: 5436301
----------------------------------------
```

**Comment by Tim55667757** [comment]
Fixed:

```
$ tksbrokerapi -t TRUR --price --depth 10
TKSBrokerAPI.py     L:1231 INFO    [2022-11-11 18:01:48,273] Current prices in order book:

Orders book actual at [2022-11-11 15:01:48] (UTC)
Ticker: [TRUR], FIGI: [BBG000000001], Depth of Market: [10]
------------------------------------------------------------
             Orders of Buyers | Orders of Sellers
------------------------------------------------------------
        Sell prices (volumes) | Buy prices (volumes)
------------------------------------------------------------
                              | 5.71 (1158)
                              | 5.7 (93508)
                              | 5.69 (112074)
                              | 5.68 (12804)
                              | 5.67 (106064)
                              | 5.66 (23593)
                              | 5.65 (1457706)
                              | 5.64 (32957)
                              | 5.63 (823159)
                              | 5.62 (1991386)
               5.61 (3351948) |
                5.6 (1780747) |
               5.59 (1354789) |
               5.58 (1167135) |
                5.57 (770161) |
                5.56 (521801) |
                5.55 (337911) |
                  5.54 (6204) |
                  5.53 (5603) |
               5.52 (1110590) |
------------------------------------------------------------
         Total sell: 10406889 | Total buy: 4654409
------------------------------------------------------------
```

**Comment by Tim55667757** [comment]
fixed in 1.5*
